src/components/Select/Select.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import Select from './Select';
+
+describe('Select', () => {
+  test('renders Select component', () => {
+    render(<Select value="Hello" />);
+
+    expect(screen.getByText('Hello'));
+  });
+});
```

Comment:
Once you've added onChange write a function that tests whether it worsk correctly.